minecraft-plugin/src/main/resources/plugin.yml

```diff
@@ -1,4 +1,4 @@
 name: Proximity_Voice_Chat
 version: '${project.version}'
 main: com.pvchat.proximityvoicechat.plugin.ProximityVoiceChat
-api-version: 1.18
+api-version: 1.18
```

Comment:
I'd recommend using editor config or something similar to avoid such pointless changes.